tests/system/suites/end_to_end_PI/t/0030_prepare_PI.test

```diff
@@ -17,21 +17,33 @@ declare OMF_TYPE_ID
 # Reads configuration setting
 source ${SUITE_BASEDIR}/suite.cfg
 
+
+# Redirects std out/err for all the following commands
+exec 7>&1                                      # Backups stdout
+exec 8>&1                                      # Backups stderr
+exec 1>>"${RESULT_DIR}/${TEST_NAME}_out.temp"
+exec 2>>"${RESULT_DIR}/${TEST_NAME}_err.temp"
+
+
 # Enables the OMF plugin
-curl -s -X PUT http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/schedule/${SCHEDULE_ID_OMF_PLUGIN} -d '{ "enabled" : true }' &>>  ${RESULT_DIR}/${TEST_NAME}.1.temp
+curl -s -X PUT http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/schedule/${SCHEDULE_ID_OMF_PLUGIN} -d '{ "enabled" : true }'
```

Comment:
What if this curl fails? See FOGL-1574

Reply:
it this command fails, 
- the  output of the command will be stored in : 
* 0030_prepare_PI_err.temp
* 0030_prepare_PI_out.temp 

- the following command '${TEST_BASEDIR}/bash/wait_creation_cfg.bash ' that wait for the creation of the configuration, will try for a while and the it will terminate the execution :  

```
FogLAMP configuration :SENT_PR_1/producerToken: not available, result :4: - N. of retries :8:
FogLAMP configuration :SENT_PR_1/producerToken: not available, result :4: - N. of retries :9:
FogLAMP configuration :SENT_PR_1/producerToken: not available, result :4: - N. of retries :10:
FogLAMP plugin :SENT_PR_1/producerToken: not available  - N. of retries :11: 



[2018-08-13 13:34:44.136718] - 0030_prepare_PI - [2018-08-13 13:34:54.628297] (10.486 seconds) - FAILED - Expect/Result MISMATCH
Total Execution Time: 23.804 seconds.
Suite End:   2018-08-13 13:34:54.638503 - INCOMPLETED
```